pkg/platform/packet/packet.go

```diff
@@ -46,6 +46,7 @@ const (
 type workerPool struct {
 	Name                  string            `hcl:"pool_name,label"`
 	Count                 int               `hcl:"count"`
+	Facility              string            `hcl:"facility,optional"`
```

Comment:
Maybe we could add a validation rule, that when this is equal to cluster facility, return an error?

Reply:
Why should that be a failure scenario? It is fine if a user wants to be explicit.

Reply:
Explicit == no-op == more configuration to understand == higher complexity IMO, hence a suggestion for returning an error.

Reply:
I don't think an error helps.

I think this behavior is nice:

- No facility in worker pool -> use the one in the controller block
- Facility in worker pool -> use that one
- Facility in worker pool is the same as in controller block -> that's fine

This is also consistent with, e.g. components: you can have default configurations but you can also specify them explicitly.

Reply:
Yeah, after looking at it one more time, this configuration would not be incorrect per-se, so we leave complexity management to the user. Marking as resolved.